docs/fedora.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Using mkosi-initrd with Fedora
+SPDX-License-Identifier: LGPL-2.1-or-later
+---
+
+# Building an initramfs image
+
+Two tools are required: `mkosi`, currently a git version is required,
+and the `mkosi-initrd` repository with configuration for `mkosi`.
+
+```bash
+git clone https://github.com/keszybz/mkosi
+git clone https://github.com/keszybz/mkosi-initrd
+cd mkosi-initrd
+```
+
+We generally want to use a cache, repeated builds will be much slower without this,
+and an output directory to avoid clutter:
+```bash
+mkdir -p mkosi.cache mkosi.output
+```
```

Comment:
I think it would be nicer to just add

```
…
OutputDirectory=mkosi.output
CacheDirectory=mkosi.cache
…
```

to mkosi.defaults. That way mkosi will create these dirs automatically when they are missing. i.e. it will not just pick them up, but create them. it's a bit like a work-around for the fact that git doesn't store empty dirs...

Benefit: it will just work, and one less step to document